UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API.FunctionalTests/FunctionalTests/EnsNotificationConfigurationTest.cs

```diff
@@ -14,12 +14,15 @@ class EnsNotificationConfigurationTest
         private EnsApiClient EnsApiClient { get; set; }
         private TestConfiguration TestConfig { get; set; }
         private D365Payload D365Payload { get; set; }
+        private string EnsToken { get; set; }
 
         [SetUp]
-        public void Setup()
+        public async Task SetupAsync()
```

Comment:
As you've made the method `async` you can now use the `async` overload of `File.ReadAllText` on line 29.

Reply:
Done required changes.